src/SDKs/_metadata/containerregistry_resource-manager.txt

```diff
@@ -0,0 +1,11 @@
+2019-07-31 22:15:17 UTC
```

Comment:
this file should be put under **eng/mgmt/mgmtmetadata**

Reply:
Done.